sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/image/automl_image.py

```diff
@@ -45,14 +60,27 @@ def __init__(
 
     @property
     def log_verbosity(self) -> LogVerbosity:
+        """Returns the verbosity of the logger.
+        :return: The log verbosity.
+        :rtype: ~azure.ai.ml._restclient.v2023_04_01_preview.models.LogVerbosity
+        """
         return self._log_verbosity
 
     @log_verbosity.setter
     def log_verbosity(self, value: Union[str, LogVerbosity]):
+        """Sets the verbosity of the logger.
+        :param value: The value to set the log verbosity to.
+        Possible values include: "NotSet", "Debug", "Info", "Warning", "Error", "Critical".
+        :type value: Union[str, ~azure.ai.ml._restclient.v2023_04_01_preview.models.LogVerbosity]
+        """
         self._log_verbosity = None if value is None else LogVerbosity[camel_to_snake(value).upper()]
 
     @property
     def limits(self) -> ImageLimitSettings:
+        """Returns the limit settings for all AutoML Image jobs.
```

Comment:
displayed incorrectly in docs, add new line before paramlist

Reply:
Check on this for all properties/methods please

Reply:
fixe